src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static Storage storage;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storage = new Storage();
+        user = new User(1234L, "login254", "1234567", 18);
+    }
+
+    @Test
+    void duplicateLoginIs_notOk() {
+        Storage.people.add(user);
+        assertThrows(InvalidUserDataException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void userLoginIs_NotOk() {
+        User actual = new User(1234L, "login", "1234567", 18);
```

Comment:
It’s enough to put the user in a global variable with standard fields, and then set the necessary fields in each test